packages/ssg/src/ssg.ts

```diff
@@ -45,31 +64,27 @@ export const ssgBuild = (options?: SSGOptions): Plugin => {
         throw new Error(`Failed to find a named export "default" from ${entry}`)
       }
 
-      console.log(`Build files into temp directory: ${tempDir}`)
+      const outDir = config.build.outDir
 
-      const result = await toSSG(app, fs, { dir: tempDir })
+      const result = await toSSG(
+        app,
+        {
+          writeFile: async (path, data) => {
+            // delegate to Vite to emit the file
+            this.emitFile({
+              type: 'asset',
+              fileName: relative(outDir, path),
+              source: data,
+            })
+          },
+          async mkdir() {},
```

Comment:
This line throws the eslint error:

<img width="679" alt="Screenshot 2024-02-17 at 8 43 31" src="https://github.com/honojs/vite-plugins/assets/10682/460f1fb2-0f39-41b2-b3c0-f76a376fbb1e">

How about writing the following to avoid it?

```ts
async mkdir() {
  return
},
```

Reply:
fixed! [`4e24768` (#80)](https://github.com/honojs/vite-plugins/pull/80/commits/4e24768c023a718900bb2cc72cfe42758d9e49b0)